scripts/TMP.sh

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+grepp LM_TMP *.cpp | grep -v './Cosas' | grep -v '// LM_TMP'
+grepp LM_TMP *.h   | grep -v './Cosas' | grep -v src/lib/logMsg/logMsg.h
```

Comment:
I find them very useful (tmp.sh and TMP.sh).
See PR comments.

If both of you think you will not use them, then we'll remove them from the PR, no probs